src/tests/ftest/pool/evict_test.py

```diff
@@ -146,43 +151,46 @@ def test_evict(self):
         by the evict
         :avocado: tags=all,pr,daily_regression,full_regression
         :avocado: tags=small
-        :avocado: tags=pool,poolevict
+        :avocado: tags=pool,pool_evict
         :avocado: tags=DAOS_5610
         """
-        pool = []
+        self.pool = []
```

Comment:
Because this test ends up invalidating the pool handle - which causes a '-1002' error when we attempt to disconnect the pool - we should not use `self.pool` in this test.
```suggestion
        # Do not use 'self.pool' for this test was it will cause -1002 errors when disconnecting.
        pool = []
```